### DISCUSSION TITLE
[oracledb] connectionAttributes is wrong for the password field (it is referencing pool instead of password)

### DISCUSSION THREAD

**Discussion by AndyNewmanCRIBL** [DefinitelyTyped/DefinitelyTyped#69278] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/oracledb/index.d.ts#L1742

line 1742
``` 
 /**
         * Specifies which previously created pool in the connection pool cache to obtain the connection from. See Pool Alias.
         */
        password?: string | undefined;
```

from an earlier commit, it should be
```
/**
         * The password of the database user. A password is also necessary if a proxy user is specified.
         */
```

[a comment by an automated account is not shown]

**Comment by AndyNewmanCRIBL** [comment]
it looks like password and pool had their descriptions swapped
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8bb11a2a03d668c2f464a492422922f609e3a4ab/types/oracledb/index.d.ts#L1385

but then pool was removed, leaving the password field still with the pool description

**Comment by ghost** [comment]
Fixed in #69379
